### PR TITLE
Remove authentication code from app and create side car containers fo…

### PR DESCRIPTION
**Pull request by SWhitlam**
…r Keycloak gatekeeper

### CODE REVIEW

File: server/app.js [hunk 1]
```diff
@@ -6,30 +6,26 @@ const csurf = require('csurf')
 const path = require('path')
 const moment = require('moment')
 const compression = require('compression')
-const passport = require('passport')
 const bodyParser = require('body-parser')
 const cookieSession = require('cookie-session')
 const sassMiddleware = require('node-sass-middleware')
-
+const auth = require('./authentication/auth')
```
Comment (nickmcmahon01) [inline server/app.js]: Is this unused?

File: helm_deploy/sentence-planning/templates/service.yaml [hunk 2]
```diff
@@ -8,12 +8,11 @@ metadata:
     release: {{ .Release.Name }}
     heritage: {{ .Release.Service }}
 spec:
-  type: NodePort
+  type: ClusterIP
```
Comment (SWhitlam) [inline helm_deploy/sentence-planning/templates/service.yaml]: I don't think we should be using NodePort here so changed to use the default ClusterIP

File: helm_deploy/values-preprod.yaml [hunk 3]
```diff
@@ -11,6 +11,19 @@ image:
   tag: latest
   pullPolicy: Always
 
+proxy:
+  image:
+    repository: quay.io/ukhomeofficedigital/nginx-proxy-govuk
+    tag: latest
```
Comment (nickmcmahon01) [inline helm_deploy/values-preprod.yaml]: Latest and IfNotPresent always ends in disaster.

File: helm_deploy/values-development.yaml [hunk 4]
```diff
@@ -11,6 +11,20 @@ image:
   tag: latest
   pullPolicy: Always
 
+proxy:
+  image:
+    repository: quay.io/ukhomeofficedigital/nginx-proxy-govuk
+    tag: latest
```
Comment (nickmcmahon01) [inline helm_deploy/values-development.yaml]: Latest and IfNotPresent always ends in disaster.

**Pull request merged**